From a73779c19768f509045473576c4c71c182ed70dc Mon Sep 17 00:00:00 2001
From: Sam Betts <sam@code-smash.net>
Date: Fri, 09 Feb 2018 15:09:07 +0000
Subject: [PATCH] Ensure we stub out the SubPort get_object function fully

Fixed a bug when running baremetal in the newton release which results in:
TypeError: get_object() got an unexpected keyword argument 'port_id'

Change-Id: I7607dadb386c27cd8730938dc073b226dd040e91
Cisco-CI-Experimental-Branch: nexus-cimc-ironic
---

diff --git a/networking_cisco/services/trunk/trunkstubs.py b/networking_cisco/services/trunk/trunkstubs.py
index 434f723..1b30a19 100644
--- a/networking_cisco/services/trunk/trunkstubs.py
+++ b/networking_cisco/services/trunk/trunkstubs.py
@@ -25,7 +25,7 @@
 
 class SubPort(object):
     @classmethod
-    def get_object(cls, context, *args):
+    def get_object(cls, context, *args, **kwargs):
         return None
 
 
